src/lib/registration/widgets/package_search.rb

```diff
@@ -201,16 +247,72 @@ def update_details
 
       # Asks the user to enable the addon
       #
-      # It omits the question if the addon is already registered or selected for registration.
-      #
       # @param addon [Addon] Addon to ask about
       def enable_addon?(addon)
+        description = Yast::HTML.Para(
+          format(
+            _("The selected package is provided by the '%{name}' module/extension, " \
```

Comment:
Looking at the screenshots, the actual module and extension names always contain the `Module` or `Extension` at the end. So the `... is provided by the 'Web and Scripting Module' module/extension, which is ...` text looks a bit weird to me. I'd remove that `module/extension`:

```suggestion
            _("The selected package is provided by the '%{name}', " \
```

So it would be just `...is provided by the 'Web and Scripting Module', which is ...`.